### DISCUSSION TITLE
Live-reload does not work with android flavor

### DISCUSSION THREAD

**Discussion by arunnac** [ionic-team/capacitor#5198] [Q&A]
If I use [Creating Environment Specific Configurations](https://capacitorjs.com/docs/guides/environment-specific-configurations) and setup flavors, the live reload works only for the default or the first flavor debug. for example app-production-debug.apk

All other flavors when run with --livereload does not work. 

weird part is, lets say you are running flavor qa (app-qa-debug.apk) even though this apk/app does not reload, but if you close this one and open the default flavor apk/app app-production-debug.apk you can see live-reload is working there.

Is there any parameter or configuration I can set to get live-reload working properly for their respective falvors?
Here is what i use to run and capacitor.config.ts has proper entries for qa for different appId and app name. I use -c=qa for environment.ts and other resources.
```
export NODE_ENV=qa
ionic build -c=qa
ionic cap sync andriod -c=qa
ionic cap run android -l --host=0.0.0.0 -c=qa
```

**Comment by jcesarmobile** [comment]
You can configure the flavor to run with android.flavor property of the capacitor config file
https://capacitorjs.com/docs/config

**Comment by arunnac** [comment]
Yes, I have adroid.falvor in the capacitor.config.ts. The "run" works fine for each flavor.

The problem is, when I try to run with live reload. 
The live reload feature works only with one of the flavor, all other flavors the app does not refresh on change like the web.

Thanks for response.
